.github/workflows/release.yml

```diff
@@ -41,7 +41,9 @@ jobs:
         ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.ORG_OSSRH_PASSWORD }}
         ORG_GRADLE_PROJECT_RELEASE_SIGNING_ENABLED: true
         ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.ORG_GPG_PRIVATE_KEY }}
-        ORG_GRADLE_PROJECT_signingInMemoryKeyId: ${{ secrets.ORG_GPG_KEY_ID }}
+        # secrets.ORG_GPG_KEY_ID contains the wrong value, so hardcode the key id as found on:
```

Comment:
I guess this is only a work-around until EF sets the `ORG_GPG_KEY_ID` variable correctly?

Reply:
Yes, I want to first make sure that the publishing actually works with that value and then ask them to fix it. I have only with `publishToMavenLocal` so far.